core/core_modules/kpm.js

```diff
@@ -210,17 +211,29 @@ var gitpull = require.safe('git-pull'),
                 fs.emptyDir(dir, function (err) {
                     cleanupCallback(); // not a lot we can do about errors here.
                 });
-            }.bind(this);
+            }.bind(this),
+                isWin = /^win/.test(process.platform);
 
             gitclone(url, dir, {}, function(err) {
                 try {
                     var kj = require.once(path.join(dir, 'kassy.json'));
-                    if (this.loadedModules[kj.name] || getModuleList()[kj.name] || getModuleList()['kpm_' + kj.name]) {
+                    kj.volitileName = kj.name;
```

Comment:
technically you could do this only for the directory name rather than the name of the module itself as well